### ISSUE TITLE
kipoi deepbind issue

### ISSUE THREAD

**Issue by drugilsberg** [IBM/depiction#66]
@phineasng @C-nit 
after a merge for a security updates with tests that we are passing we are getting this error:
Instructions for updating:
```
If using Keras pass *_constraint arguments to layers.

Shutting down kernel

---------------------------------------------------------------------------

TypeError                                 Traceback (most recent call last)

/opt/conda/bin/depiction-models-download in <module>

     23     _ = CellTyper(cache_dir=args.cache_dir)

     24     # DeepBind

---> 25     _ = DeepBind(cache_dir=args.cache_dir)

     26     # PaccMann

     27     # NOTE: here data_type is not relevant

TypeError: __init__() got an unexpected keyword argument 'cache_dir'
```
I think the kipoi model update was not reflected in this script.

**Comment by drugilsberg** [comment]
How do you suggest to proceed @C-nit ? We change the import in the script (to ensure compatibility of the docker images), or we change the class constructor.

**Comment by C-nit** [comment]
Depends if the docker image should still support the cli version of deepbind.
This would simply require changing the import.

Instead or additionally we should download the specific TF models from kipoi used in the notebook during the image build instantiating each of them once. I think we use only two in the notebook.

I'd be for removing the cli version cache, but without internet connection the containers will only support these two models.

**Comment by C-nit** [comment]
Though it would be tough to use the old one now anyway since the instructions changed to use the kipoi version.

**Comment by drugilsberg** [comment]
Exactly, not easy to decide, maybe the script should still download the old version, in case someone runs one of the notebooks from PyPharma.

**Comment by drugilsberg** [comment]
Closed in #67
